wgs/paired-fastq-to-unmapped-bam/README.md

```diff
@@ -0,0 +1,7 @@
+The workflow in this repository is the Paired FastQ to Unmapped BAM workflow provided by the Broad Institute.
+
+A copy of the paired-fastq-to-unmapped-bam.wdl file was made on February 23, 2018 from the Broad's github repo:
+
+https://github.com/gatk-workflows/seq-format-conversion/blob/413fca8295d44ea0a6c147bab6663d7cfdd1d808/paired-fastq-to-unmapped-bam.wdl
+
+No changes were required to the workflow for use in AMP PD.
```

Comment:
No changes were made to the workflow for AMP PD WGS single sample processing.

Reply:
This change has been made